scripts/zones/Lufaise_Meadows/mobs/Padfoot.lua

```diff
@@ -11,12 +11,19 @@ local ID = require("scripts/zones/Lufaise_Meadows/IDs")
 -----------------------------------
 local entity = {}
 
-entity.onMobSpawn = function(mob)
-    if mob:getID() == ID.mob.PADFOOT[GetServerVariable("realPadfoot")] then
-        mob:setDropID(2911)
-    else
-        mob:setDropID(1972)
-    end
+entity.onMobInitialize = function(mob)
+    mob:addListener("ITEM_DROPS", "ITEM_DROPS_PADFOOD", function(mobArg, loot)
+        if mob:getID() == ID.mob.PADFOOT[GetServerVariable("realPadfoot")] then
+            loot:addGroup(xi.loot.rate.GUARANTEED,
+            {
+                { item = xi.items.ASSAILANTS_RING, weight = 75 },
```

Comment:
we're only using 0-100 for the % rates? I'd like to be consistent with the sql pattern and have the extra decimal point precision if only to match it.

Reply:
That makes sense. I changed this to be 750 and 250 to match sql 1000 weights.